src/shell/Shell.css

```diff
@@ -8,7 +8,8 @@
     }
 
     & .content {
-        flex: 1 0 auto;
-        background: yellow;
+        width: calc(100% - 8rem);
```

Comment:
I think all the positioning of the sidebar and content should be here, including the position fixed. Also, we no longer need the display flex on the shell and flex 0 0 auto on the sidebar.